### PR TITLE
Pick an openSUSE distribution

### PR DESCRIPTION
**Pull request by Nycticorax**
Signed-off-by: Nycticorax <mrnycticorax@gmail.com>

### CODE REVIEW

**Review by daviddyess — COMMENTED** [review]
I like this introduction better than my original and I like the mention of microOS as an option.

File: project/docs/desktop_choice.md [hunk 1]
```diff
@@ -1,3 +1,41 @@
...
+Tumbleweed is a _rolling-release_ distribution: instead of having fixed points where you will be offered to perform a system upgrade (think of Windows, Ubuntu, or macOS, where each numbered version leads to the next), you will be frequently offered to upgrade multiple components of your system.
+
+??? info
+    Those components are called _packages_, because they "pack" the programs and resources making up most of your system, along with tools to install them). "Upgrading" means "Upgrading packages" (i.e. those that constitute you Linux distribution).
+
+Even though Tumbleweed uses a rolling-release model, it is very stable: the packages are thoroughly tested and tried, and you can always retreat to the previous state of Tumbleweed (in jargon: "switching to the previous snapshot". Snapshots are explained [here]()).
+
+What you have to bear in mind, however, is that Tumbleweed requires a reliable internet connection and a healthy hard drive to work as designed. It also requires that you sometimes take interest in the updating process. This means that you should probably not pick Tumbleweed if any of these conditions holds for you:
+
+### Leap
```
[the remark this answers is not in the document]
Comment (Nycticorax) [reply project/docs/desktop_choice.md]: Fixed, thanks you David!

**Comment by apinter** [comment]
IMHO these are 2 separate documents. If we're talking about distros lets talk about distros, if we're talking about DE/WM lets focus on that. My two cents.

**Comment by Nycticorax** [comment]
> IMHO these are 2 separate documents. If we're talking about distros lets talk about distros, if we're talking about DE/WM lets focus on that. My two cents.

Is it better if we rename the document "How to pick the image you're going to download"? Doesn't it bring the unity of matters you're after?

**Pull request merged**